src/lib/decorators.ts

```diff
@@ -227,15 +227,16 @@ export function query(selector: string, cache?: boolean) {
       configurable: true,
     };
     if (cache) {
-      const key = typeof name === 'symbol' ? Symbol() : `__${name}`;
+      const key = name || (protoOrDescriptor as ClassElement).key;
+      const cacheKey = typeof key === 'symbol' ? Symbol() : `__${key}`;
```

Comment:
How about we just use a symbol if the name is undefined?

Reply:
I think we should either always use a symbol, or keep it as I have it now for consistency.
Otherwise, the `cacheKey` is different based on the compiler you use.

I'm fine with always using a symbol as cacheKey, I guess `__${name}` was chosen for debugging purposes?

Reply:
We should only use a Symbol if the name is a Symbol, since we don't require the Symbol polyfill currently.